setup.py

```diff
@@ -274,7 +274,6 @@ def finalize_options(self):
 
 data_files = [
     (ETC + "/cloud", [render_tmpl("config/cloud.cfg.tmpl", is_yaml=True)]),
-    (ETC + "/cloud/clean.d", glob("config/clean.d/*")),
```

Comment:
The result of no longer delivering this FILE/directory will break external consumers like subiquity which expect the /etc/cloud/clean.d directory to exist. [Subiquity injects supplemental clean.d files](https://github.com/canonical/subiquity/blob/main/subiquity/models/subiquity.py#L453) so when cloud-init is cleaned from live desktop/serve deployed images, the datasource_list is reset to defaults and cloud-init discovery is "re-enabled". We either need to either:
 1. ensure this empty /etc/cloud/clean.d directory still exists
 2. or continue to deliver the /etc/cloud/clean.d/README but provide a param to runparts to be silent on non_exe files, or allow for providing a ignore_files list param which which skip the expected README

Reply:
Originally this PR just removed the README file. Brett mentioned `setup.py` in his review, but I don't think anything actually needs to be changed, so we should just remove the change on this line.

Reply:
@blackboxsw I think option 1 is better; any documentation, e.g. README should be on our documentation.

Reply:
Works from me since we now have top-level docs on ReadTheDocs for this. thanks